### ISSUE TITLE
Create group: 403 forbidden

### ISSUE THREAD

**Issue by Asher256** [gpocentek/python-gitlab#130]
I tried to create a group but the Gitlab module failed:
```
In [5]: result = git.groups.create({'name': 'test', 'path': 'test'})
2016-07-18 13:00:27,850 Resetting dropped connection: git.gameloft.org
---------------------------------------------------------------------------
GitlabCreateError                         Traceback (most recent call last)
//home/asher/.puppet/modules/gm_gitlab/files/helpers/add_project.py in <module>()
----> 1 result = git.groups.create({'name': 'test', 'path': ''})

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/objects.py in create(self, data, **kwargs)
    139         if not self.obj_cls.canCreate:
    140             raise NotImplementedError
--> 141         return self.obj_cls.create(self.gitlab, data, **args)
    142 
    143     def delete(self, id, **kwargs):

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/objects.py in create(cls, gl, data, **kwargs)
    354 
    355         obj = cls(gl, data, **kwargs)
--> 356         obj.save()
    357 
    358         return obj

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/objects.py in save(self, **kwargs)
    324             self._update(**kwargs)
    325         else:
--> 326             self._create(**kwargs)
    327 
    328     def delete(self, **kwargs):

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/objects.py in _create(self, **kwargs)
    309             raise NotImplementedError
    310 
--> 311         json = self.gitlab.create(self, **kwargs)
    312         self._set_from_dict(json)
    313         self._from_api = True

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/__init__.py in create(self, obj, **kwargs)
    595                 "Can't connect to GitLab server (%s)" % e)
    596 
--> 597         raise_error_from_response(r, GitlabCreateError, 201)
    598         return r.json()
    599 

//home/asher/.puppet/modules/gm_gitlab/files/helpers/gitlab/exceptions.py in raise_error_from_response(response, error, expected_code)
    146     raise error(error_message=message,
    147                 response_code=response.status_code,
--> 148                 response_body=response.content)

GitlabCreateError: 403: 403 Forbidden
```

Any idea Gauvain? 

Thank you!

Asher256

**Comment by Asher256** [comment]
I found the solution here https://gitlab.com/gitlab-org/gitlab-ce/issues/4259 . The issue was related to Gitlab, not to the Python module.

Stephan H.: "Found the problem. I had the impression that an admin user could per se create anything in Gitlab. I did not realize that I need to explicitly enable the "Can create group" flag for the user. After ticking the checkbox I am able to create a Group via the respective API endpoint as expected."

**Comment by Asher256** [comment]
I found the solution here https://gitlab.com/gitlab-org/gitlab-ce/issues/4259 . The issue was related to Gitlab, not to the Python module.

Stephan H.: "Found the problem. I had the impression that an admin user could per se create anything in Gitlab. I did not realize that I need to explicitly enable the "Can create group" flag for the user. After ticking the checkbox I am able to create a Group via the respective API endpoint as expected."

**Comment by Asher256** [comment]
I found the solution here https://gitlab.com/gitlab-org/gitlab-ce/issues/4259 . The issue was related to Gitlab, not to the Python module.

Stephan H.: "Found the problem. I had the impression that an admin user could per se create anything in Gitlab. I did not realize that I need to explicitly enable the "Can create group" flag for the user. After ticking the checkbox I am able to create a Group via the respective API endpoint as expected."
